scripts/page_browse.js

```diff
@@ -0,0 +1,247 @@
+/* global $ pageBrowserData dpData splitControl mode */
```

Comment:
We need to change `dpData` to something more specific as dpData could mean anything. `proofIntData`?